docs/layouts/_shortcodes/csp.html

```
@@ -1,2 +1,2 @@
-<p><strong>Placeholder Value:</strong>  <code>{{ hugo.Data.misc.csp.nonce }}</code></p>
+<p><strong>Placeholders:</strong> <code>{{ hugo.Data.misc.csp.nonce }}</code> (per-request nonce), <code>{{ hugo.Data.misc.csp.oidc_client_logo_uris }}</code> (per-request OIDC consent client logo origin, empty on non-consent routes)</p>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)